test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/project.json

```diff
@@ -7,7 +7,7 @@
     "Microsoft.AspNetCore.Server.Kestrel": "1.0.0-*",
     "Microsoft.AspNetCore.Server.Kestrel.Https": "1.0.0-*",
     "Microsoft.AspNetCore.Testing": "1.0.0-*",
-    "Newtonsoft.Json": "8.0.3",
+    "Newtonsoft.Json": "9.0.1-beta1",
```

Comment:
Great! Once xunit.assert gets ported to net standard, then we can just get rid off the entire import chain! :smile: